src/helpers/constants.js

```diff
@@ -17,6 +17,13 @@ export const STRATEGIES = {
   NO_PREFIX: STRATEGY_NO_PREFIX
 }
 
+const REDIRECT_ROOT = 'root'
+const REDIRECT_NO_PREFIX = 'no_prefix'
+export const REDIRECTIONS = {
+  ROOT: REDIRECT_ROOT,
+  NO_PREFIX: REDIRECT_NO_PREFIX
+}
```

Comment:
Would like the `redirectOn` to be reflected in the constant name also. So something like:

```suggestion
const REDIRECT_ON_ROOT = 'root'
const REDIRECT_ON_NO_PREFIX = 'no_prefix'
export const REDIRECT_ON_OPTIONS = {
  ROOT: REDIRECT_ON_ROOT,
  NO_PREFIX: REDIRECT_ON_NO_PREFIX
}
```